examples/placegoose/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import { instrument } from "@fiberplane/hono-otel";
+import { cloudflareRateLimiter } from "@hono-rate-limiter/cloudflare";
+import { Hono } from "hono";
+import { HTTPException } from "hono/http-exception";
+
+import * as routes from "./routes";
+
+type AppType = {
+  Variables: {
+    rateLimit: boolean;
+  };
+  Bindings: {
+    RATE_LIMITER: RateLimit;
+  };
+};
+
+const app = new Hono<AppType>();
+
+app.use(
+  cloudflareRateLimiter<AppType>({
+    rateLimitBinding: (c) => c.env.RATE_LIMITER,
+    keyGenerator: (c) => c.req.header("cf-connecting-ip") ?? "",
+  }),
+);
+
```

Comment:
added global cors. could you clarify how that connects to rate limiting?

Reply:
yeah so from the docs, CF rate limiter operates on unique key per cloudflare location

since we're dealing with anonymous requests (no user session or api key) then we'd probably want to do something based on IP address. 

we can be quite generous since this is a low-cost application. maybe 100 per minute would be okay per ip address?